src/pg_cron.c

```diff
@@ -530,8 +618,13 @@ PgCronLauncherMain(Datum arg)
 
 		AcceptInvalidationMessages();
 
-		if (!CronJobCacheValid)
+		if (!CronJobCacheAndEreportValid)
```

Comment:
It might be nicer to have 2 separate flags, rather than one awkwardly named one. 

e.g. CronJobCacheValid, ReloadConfig

also because I now noticed that CronJobCacheValid is not just set by SIGHUP